Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM puckel/docker-airflow:1.10.6
```

Comment:
Should we use the official docker image distributed by Apache for Airflow instead? See [`apache/airflow`](https://hub.docker.com/r/apache/airflow)

Reply:
Looks like using `puckel/docker-airflow` might be the more "stable" image to us. Disregard my comment above :)